src/IssueViz.Security/Commands/Constants.cs

```diff
@@ -25,5 +25,8 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.Commands
     internal static class Constants
     {
         public static readonly Guid CommandSetGuid = new Guid("97856422-20A2-4DB5-A468-1BAA9B6EEC38");
+
+        public const int HotspotsToolWindowCommandId = 0x0100;
+        public const int TaintToolWindowCommandId = 0x0101;
```

Comment:
Now that we have multiple commands it's convenient to have the id constants in the same file - makes it easier to check that we are using the same values as in the `vsct`.